firebird/__init__.py

```diff
@@ -1,15 +1,76 @@
-CASSANDRA_CONTACT_POINTS = os.getenv('CASSANDRA_CONTACT_POINTS', '0.0.0.0')
-CASSANDRA_USER = os.getenv('CASSANDRA_USER')
-CASSANDRA_PASS   = os.getenv('CASSANDRA_PASS')
+import os
+import sys
+import logging
+import numpy as np
+from datetime import datetime
 
-LOG_LEVEL = ''
+AARDVARK_HOST             = os.getenv('AARDVARK_HOST', 'localhost')
+AARDVARK_PORT             = os.getenv('AARDVARK_PORT', '5678')
+AARDVARK_SPECS            = os.getenv('AARDVARK_SPECS', '/landsat/chip-specs')
+AARDVARK_SPECS_URL        = ''.join([AARDVARK_HOST, ':', AARDVARK_PORT, AARDVARK_SPECS])
 
-SPARK_MASTER = os.getenv('SPARK_MASTER', 'spark://localhost:7077')
+CASSANDRA_CONTACT_POINTS  = os.getenv('CASSANDRA_CONTACT_POINTS', '0.0.0.0')
+CASSANDRA_USER            = os.getenv('CASSANDRA_USER')
+CASSANDRA_PASS            = os.getenv('CASSANDRA_PASS')
+CASSANDRA_KEYSPACE        = os.getenv('CASSANDRA_KEYSPACE', 'lcmap_changes_local')
+
+LOG_LEVEL                 = os.getenv('FIREBIRD_LOG_LEVEL', 'INFO')
+
+SPARK_MASTER              = os.getenv('SPARK_MASTER', 'spark://localhost:7077')
+SPARK_EXECUTOR_IMAGE      = os.getenv('SPARK_EXECUTOR_IMAGE')
+SPARK_EXECUTOR_CORES      = os.getenv('SPARK_EXECUTOR_CORES', 1)
+SPARK_EXECUTOR_FORCE_PULL = os.getenv('SPARK_EXECUTOR_FORCE_PULL', 'false')
+
+LCMAP_PRODUCT_DICT = {
```

Comment:
LCMAP_PRODUCT_DICT should go away, that was from an earlier attempt at letting users specify what products they wanted via string args